docs/JSON-RPC.md

```diff
@@ -186,6 +186,22 @@ Results:
 | result.clients | array | The client list. See jamulusclient/clientListReceived for the format. |
 
 
+### jamulusclient/pollServerList
+
+Requests the server list from a specified directory.
```

Comment:
Sorry for my forgetfulness here - you need to update the python script I mentioned below and not edit the file directly

Reply:
I have run the script and pushed the changes.